Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/test/DAOIntegrationTest.java

```diff
@@ -0,0 +1,30 @@
+/**
+ *
+ */
+package gov.hhs.fha.nhinc.test;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+/**
+ * Loads up the testing context for Spring, and sets the nhinc properties directory to the test folder. </br>
+ * Hibernate will try and load up the config files from "/CONNECTCoreLib/src/test/resources/config/hibernate". These
+ * should match the general setup, with the exception of changing the JNDI name to use a JDBC connection.
+ *
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(locations = { "/config/CONNECT-testcontext.xml" })
+public class DAOIntegrationTest {
+    {
+        // We need to set this property so the PropertyAccessor class doesnt complain and error out.
+        System.setProperty("nhinc.properties.dir", System.getProperty("user.dir") + "/src/test/resources/");
+    }
+
+    @Test
+    public void contextLoads() {
```

Comment:
this test doesn't have anything

Reply:
If the Spring Context does not load, it will throw an exception, which will cause the test to fail. That's the point of the test.

Reply:
You can use Beforeclass annotation so that it can be loaded.